src/renderer/components/+preferences/add-helm-repo-dialog.tsx

```diff
@@ -107,7 +110,7 @@ export class AddHelmRepoDialog extends React.Component<Props> {
 
   async addCustomRepo() {
     try {
-      await HelmRepoManager.addCustomRepo(this.helmRepo);
+      await HelmRepoManager.getInstance().addRepo(this.helmRepo);
       Notifications.ok(<>Helm repository <b>{this.helmRepo.name}</b> has added</>);
```

Comment:
noticed this while testing:
```suggestion
      Notifications.ok(<>Helm repository <b>{this.helmRepo.name}</b> has been added</>);
```